### ISSUE TITLE
purpose -> scope / dct:abstract

### ISSUE THREAD

**Issue by heidivanparys** [SEMICeu/CAMSS_CSSV#8]
Please consider renaming purpose to scope. "scope" is used in a lot in different organisations. See e.g. the specifications of ISO, Open Geospatial Consortium (OGC), the Object Management Group, European Telecommunication Standards Institute, the INSPIRE specifications, etc.

Should Scope perhaps be its own class, as both "cssv:Purpose" and "cssv:PurposeID" are present?

Another option might be to use dct:abstract instead (and remove cssv:PurposeID): see https://www.iso.org/sites/directives/current/part2/index.xhtml#_idTextAnchor169

> The Scope shall be succinct so that it can be used as a summary for bibliographic purposes, for example, as an abstract.

**Comment by isa-camss** [comment]
Thank you for the input @heidivanparys. After reading your comment our reaction would be:

- We now see that "purpose" may be inconvinient because it could imply that you restrict the use of the Specification and, to some extent specification should be unaware of its possible uses. The term "scope" would fit better to the concept of area of requirements addressed by the specification. 

- About whether the scope should a class or a property: We tend to see it as a class rather than a property because, if it is uniquely identified, then it can be registered/repositoried and thus findable and reusable. Additionally, it opens a door to extend the "scope" in future application profiles and allow the linking of other classes to the "scope", and therefore to any descendant of the "scope". 

- The idea of using dct:abstract is also interesting but, we tend to see it as an attribute of the class "Specification", additionally having a separate class "Scope" with its own description.

In conclusion, action points: 

1. Add the class "Scope" with the attributes "ID"and "Description". Would you see the need fora type "code" to categorise scopes?
2. Add the attribute "dct:abstract" to the class "Specification".

**Comment by heidivanparys** [comment]
> Would you see the need fora type "code" to categorise scopes?

No, I dont' think that is possible. Usually the scope is at least few sentences long, and only intended for humans.

E.g.

> 
> The objective of this public consultation is to produce a very first release of a “Core Standards and Specifications Vocabulary” (CSSV) which will be used for the development of the new ELIS release and for any need of information exchange related to standards and specifications amongst software solutions. The update of this release of the CSSV will be based on requests for change coming from stakeholders interested in the Vocabulary.
> The Core Standards and Specifications (CSSV) specified in this document has been developed taking in inputs from different sources, namely the works developed for the ISA2’s CAMSS Action, interested MS, the Semantic Interoperability Community (SEMIC) action of the ISA2 Programme and IT consultants working for the European Commission.

What perhaps could be useful is what in OGC is called "standardization target", defined as "entity to which some requirements of a standard apply" (in the context of a specification, the term would be "specification target").

See e.g. http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_conformance , where the standardization target is "Web APIs", or IndoorGML, where the standardization target is "IndoorGML instance documents".

See more in https://www.opengeospatial.org/standards/modularspec

**Comment by EmidioStani** [comment]
This repository is going to be archived, thus I am closing this issue.
